pkg/obfuscator/ip.go

```diff
@@ -58,12 +58,12 @@ func (o *ipObfuscator) replace(s string) string {
 				continue
 			}
 
-			cleaned := strings.ReplaceAll(m, "-", ".")
+			cleaned := strings.ToUpper(strings.ReplaceAll(m, "-", "."))
```

Comment:
not sure what toUpper does with numbers, what's the rationale to add that?

Reply:
IPv6 addresses

Reply:
dang, makes sense. another good reason to split this up. Is there no test case that needs adaptation to this?

Reply:
I added 2 new test cases